qutebrowser/browser/qutescheme.py

```diff
@@ -158,6 +163,86 @@ def qute_bookmarks(_url):
     return 'text/html', html
 
 
+@add_handler('history')
+def qute_history(url):
+    """Handler for qute:history. Display history."""
+    # Get current date from query parameter, if not given choose today.
+    curr_date = datetime.date.today()
+    try:
+        query_date = QUrlQuery(url).queryItemValue("date")
+        if query_date:
+            curr_date = datetime.datetime.strptime(query_date, "%Y-%m-%d")
+            curr_date = curr_date.date()
+    except ValueError:
+        log.misc.debug("Invalid date passed to qute:history: " + query_date)
```

Comment:
I think this should be a message shown directly in the browser as users might modify the URL manually and get no feedback on errors this way.